### PR TITLE
OSDOCS-9449: adds RHEL 9.4 to Microshift 4.16 relnotes

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.16 only

Issue:
[OSDOCS-9449](https://issues.redhat.com/browse/OSDOCS-9449)

Link to docs preview:
[Release notes RHEL](https://75540--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-16-release-notes.html#microshift-4-16-rhel)

SME review:
- [x] SME has approved this change.

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by aravipra** [comment]
@copejon review and LGTM required

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 1]
```diff
@@ -30,7 +30,7 @@ This release adds improvements related to the following components and concepts.
 //L3 major categories with features in each as L4s
 [id="microshift-4-16-rhel"]
 === {op-system-base-full}
-//* {microshift-short} now runs on {op-system-base-full} 9.4
+* {microshift-short} now runs on {op-system-base-full} 9.4
```
Comment (tmalove) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:33]:
```suggestion
* {microshift-short} now runs on {op-system-base-full} 9.4.
```

**Review by tmalove — APPROVED** [review]

File: microshift_release_notes/microshift-4-16-release-notes.adoc (hunk 1 above)
Comment (aravipra) [reply microshift_release_notes/microshift-4-16-release-notes.adoc]: Done

**Review by jeana-redhat — APPROVED** [review]
LGTM
/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**